geonode/qgis_server/tests/test_views.py

```diff
@@ -325,6 +325,24 @@ def test_ogc_specific_layer(self):
         uploaded.delete()
         vector_layer.delete()
 
+    @on_ogc_backend(qgis_server.BACKEND_PACKAGE)
+    def test_upload_geojson(self):
+        """Test we can upload a geojson layer and check bbox"""
+        file_path = os.path.realpath(__file__)
+        test_dir = os.path.dirname(file_path)
+        # a geojson sample is taken from http://geojson.org
+        geojson_file_path = os.path.join(test_dir, 'data', 'point.geojson')
```

Comment:
Can you also test for FeatureCollection geojson data?

Reply:
just created a new issue: https://github.com/kartoza/geonode/issues/477
i think the unittest for FeatureCollection can be added later after the issue been addressed.